recipes/intel-opencl-runtime/build.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -ex
+
+src="$SRC_DIR/$PKG_NAME"
+mkdir -p $PREFIX/lib/intel-ocl-cpu
+
+# Move to intel-ocl-cpu to avoid clashes with intel-opencl-clang
+cp -rv "$src/lib/"* "$PREFIX/lib/intel-ocl-cpu/"
+
+# Use conda-forge's opencl loader
+rm -rf $PREFIX/lib/intel-ocl-cpu/libOpenCL.*
```

Comment:
@tomashek, we are deleting the ICD loader that `opencl_rt` comes with and use own ICD loader.